doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
+or the **"CNI multiplexer/meta-plugin"** if it is deployed. For example,
+the integration of SDN solutions required by `req.inf.ntw.05` is enabled
+via CNI integration.
+- **Service Mesh**: The well known service meshes are "application service meshes" that address and
+interact with the application layer 7 protocols (eg.: HTTP) only. Therefore,
+their support is not required in this architecture, as these service meshes are
+outside the scope of the infrastructure layer of the CNTT stack.
 
-A CNI metaplugin/CNI multiplexer has the capability to attach several
-interfaces, using different other CNI plugins, to a pod. Note that the different
-network characteristics of the interfaces might require different networking
-technologies, which would potentially require different CNI plugins.
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
 
-To comply with `req.inf.ntw.08`, inter node communication must be served by a
-CNI plugin which complies with the default [Kubernetes networking assumptions](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model).
+<!--The above diagram is maintained here: https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
 
 There are two types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
-> Note: An infrastructure can provide the possibility to use SR-IOV with DPDK as
-an additional feature and still be conformant with CNTT.
-
-> Editors note: The possibility to SR-IOV for DPDK is under discussion.
-
-The integration of SDN solutions required by `req.inf.ntw.05` should be enabled
-via their respective CNI integration.
 
-> Note: An SDN solution can manage the pod networks via the Kubernetes API or
-can have a CNI integration.
+| Requirement | Networking Solution with Multus | Networking Solution with DANM |
+|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
```

Comment:
1. \<broken record> I don't think this table is needed as we don't compare other CNI plugins for the same solution.
2. Tungsten Fabric should be included even though it is not a multiplexer but rather a default CNI that doesn't require a multiplexer to support multiple network connections -- @sukhdevkapur had stated that in a couple of meetings
3.  The support for DPDK and SR-IOV should be added as requirements to this Table: [infra.net.acc.cfg.001](https://github.com/cntt-n/CNTT/blob/tomkivlin-ra2-networking/doc/ref_arch/kubernetes/chapters/chapter02.md#223-cloud-infrastructure-software-profile-requirements)

Reply:
@pgoyal01 
1. At the moment we define two options for CNI multiplexers DANM and Multus and as there are differences between them we need to describe these differences somewhere. I do not see why is this a returning problem. 
2. Please create a pr if you think that Tungsten Fabric should be specifically mentioned somewhere in the document.
3. Agreed

Reply:
@CsatariGergely 
1. We should stop treating one or the other of our example multiplexers as the reference. Both Multus and DANM are CNI multiplexers.
2. Just because DANM has chosen to be more than a multiplexer doesn't mean that Multus as a multiplexer has gaps.
3. We have mentioned Multus and DANM as examples. Tomorrow if we add CNI-Genie as an example, would we create DANM gaps because CNI-Genie has incorporated some other functions.
4. We cannot add Tungsten Fabric (TF) as a multiplexer. When TF is used as the default CNI it does away with the need for a multiplexer.
5. In many CNTT documents we use example solutions but do not undertake to compare them.

Reply:
This will be covered by Issue #2134

Reply:
I have added a TF based solution and NSM based solution to the table - will need some help to fill in some parts / check they're correct (@pgoyal01, @sukhdevkapur, @fkautz).

Reply:
Sorry @pgoyal01 I didn't see that issue before I did the latest commit.